### ISSUE TITLE
modes_spec, "yellow flash affects all signal groups" fails due to undocumented timeout value yellow_flash.

### ISSUE THREAD

**Issue by sveitech** [rsmp-nordic/rsmp_validator#363]
This pull request introduced a new timeout value called yellow_flash: https://github.com/rsmp-nordic/rsmp_validator/commit/1df95a649b2474e61e025b3a384b6bb7ae18f7cd

The test in the title breaks if the field is not added to the config.

Consider having default values for new fields introduced to the config.

The documentation should also be updated. The field is not documented on https://rsmp-nordic.github.io/rsmp_validator/config/

**Comment by emiltin** [comment]
Hi,
I've added the helper `Validator.get_config()` to fetch a config in the PR #364.

If the config is missing it will stop with an error, telling you that it's missing. 

It also support providing a default value, which will be used if the config is missing. but it's not currently used in any test. In general I tihnk it's better to avoid default values, and instead rely on an explicit config for the equipment you're testing.

I've updated the documentation on  https://rsmp-nordic.github.io/rsmp_validator/config/ to include `yellow_flash`.
